dts/arm/nxp/nxp_k6x.dtsi

```diff
@@ -41,6 +41,7 @@
 		uart-5 = &uart5;
 		usbd-0 = &usbd;
 		watchdog0 = &wdog;
+		rtc-0 = &rtc0;
```

Comment:
please move this up in the list to preserve alpha order

Reply:
Fixed.